### PR TITLE
Feat: Adding Kubevirt-Tutorial to Prow config allowing repo management

### PR DESCRIPTION
**Pull request by jparrill**
**Added Kubevirt-Tutorial repo to Prow CI/CD**

- Added Config to manage AutoMerges on Kubevirt-Tutorial repo
- Added Plugins:
  - approve
  - lgtm
  - owners-label
  - config-updater
  - skip
  - override
  - trigger

- Added Labels for this concrete repo:
  - kind/lab-updates
  - kind/lab-deployment
  - kind/new-lab

### CODE REVIEW

**Comment by jparrill** [comment]
I will need-rebase because of this: https://github.com/kubevirt/project-infra/pull/122

**Comment by jparrill** [comment]
/do-not-merge/work-in-progress

File: github/ci/prow/files/plugins.yaml [hunk 1]
```diff
@@ -62,6 +62,15 @@ plugins:
   - override
   - trigger
 
+  kubevirt/kubevirt-tutorial:
+  - approve
+  - lgtm
+  - owners-label
+  - config-updater
```
Comment (rmohr) [inline github/ci/prow/files/plugins.yaml]: not needed

File: github/ci/prow/files/plugins.yaml [hunk 2]
```diff
@@ -62,6 +62,15 @@ plugins:
...
+  - skip
```
Comment (rmohr) [inline github/ci/prow/files/plugins.yaml]: skip and override are now enabled for the whole org.

**Comment by rmohr** [comment]
/lgtm
/approve

Don't forget to allow squash merges on your github repos.

**Pull request merged**